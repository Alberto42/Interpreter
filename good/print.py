x = 3
y = "str"
z = False
a = [x,y,z]
b = (x,y,z)
print(x)
print(y)
print(z)
print(a)
print(b)

