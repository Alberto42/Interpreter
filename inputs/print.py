x = 3
y = "str"
z = False
print(x)
print(y)
print(z)
