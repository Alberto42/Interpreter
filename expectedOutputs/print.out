Ident "z" = VBoolean False
Ident "y" = VString "str"
Ident "x" = VInt 3

3
str
False

